src/main/common/utils/freeMediationUtil.ts

```diff
@@ -35,29 +34,14 @@ export class FreeMediationUtil {
     } else if (mediationDraft.canWeUse) {
       if (mediationDraft.canWeUse.option === FreeMediationOption.YES) {
         if (!claim.isResponseSubmitted() && draft) {
-          return draft.defendantDetails.mobilePhone.number || undefined
+          return draft.defendantDetails.phone.number || undefined
         } else {
-          return claim.claimData.claimant.mobilePhone || mediationDraft.canWeUse.mediationPhoneNumber
+          return claim.claimData.claimant.phoneNumber || mediationDraft.canWeUse.mediationPhoneNumber
         }
       } else {
         return mediationDraft.canWeUse.mediationPhoneNumber
       }
     }
     return undefined
   }
-
-  static getMediationContactPerson (claim: Claim, mediationDraft: MediationDraft, draft?: ResponseDraft): string {
```

Comment:
why is this being removed?

Reply:
Unused

Reply:
it is used on the mediation branch. I pulled it out into a PR to make that one more manageable

Reply:
Re-added